lib/src/utils/constants.dart

```diff
@@ -0,0 +1,7 @@
+const int harden = 0x80000000;
+const int claCardano = 0xD7; // CLA for Cardano
+const int insDeriveAddress = 0x11; // INS for Derive Address
```

Comment:
This should be included already in the enum for instructions. It should be removed.

Reply:
cardano_ledger_operation.dart will be deleted so all will be constants